Seminar02/Week2_Simeon.cpp

```diff
@@ -0,0 +1,34 @@
+
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    /*cout << "Exercise 1: \n";
+    int year;
+    cin >> year;
+    if (year % 4 == 0) cout << "The year is visokosna!\n";
```

Comment:
What is a leap year? To be a leap year, the year number must be divisible by four – except for end-of-century years, which must be divisible by 400. This means that the year 2000 was a leap year, although 1900 was not. 2024, 2028, 2032 and 2036 are all leap years.